pub_sub/lib/pubsub.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Pubsub do
+  use Application
```

Comment:
I don't understand why this is here: you don't seem to be creating an application.